py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1049,7 +1058,7 @@ def aten_ops_permute(
 
 
 def to_copy_dtype_validator(
-    placeholder_only: bool, settings: CompilationSettings = None
+    placeholder_only: bool, settings: Optional[CompilationSettings] = None
```

Comment:
why the change to optional here ?

Reply:
Lint pointed it out. I think it's because the default is None